docs/getting_started.rst

```diff
@@ -30,7 +30,8 @@ file, place it in the ``cfme_tests`` repository (along ``conftest.py``):
 
   . ./.cfme_tests/bin/activate
   python scripts/disable-bytecode.py
-  PYCURL_SSL_LIBRARY=nss pip install -Ur ./requirements.txt
+  pip install --upgrade pip
+  PYCURL_SSL_LIBRARY=nss pip install -Ur ./requirements.txt --no-cache-dir
```

Comment:
For 'no-cache-dir', since this would only possibly increase the time for the initial setup and could resolve possible issues beforehand for people less experienced or whom skim the instructions. I still think it should be added.

Then for the pip part. I just tested this since I did not have useful data.

- New untouched venv  pip version - 1.4.1

- Post pip upgrade version - 8.1.2

- System pip version - 7.1.0